src/RBCXManager.cpp

```diff
@@ -155,6 +160,20 @@ void Manager::sendToCoproc(const CoprocReq& msg) {
     xTaskNotify(m_keepaliveTask, 0, eNoAction);
 }
 
+void Manager::coprocFwVersionAssert(uint32_t minVersion, const char* name) {
+    if (m_coprocFwVersion.number == 0
+        && xTaskGetTickCount() < pdMS_TO_TICKS(50)) {
+        vTaskDelay(25);
```

Comment:
Zde je čekání, protože ještě `m_coprocFwVersion.number` nebylo v jiném tasku nastaveno? Pokud ano, možná by se hodil komentář.

Reply:
Nakonec jsem to změnil, to čekání nedává moc smysl :)